spec/features/content_items_spec.rb

```diff
@@ -0,0 +1,20 @@
+require 'rails_helper'
+require 'features/pagination_spec_helper'
+
+RSpec.feature "Content Items List", type: :feature do
+  background do
+    @content_items = create_list :content_item_with_organisations, 3
+  end
+
+  scenario "the user sees a list of all unfiltered content items and clicks through to an organisation" do
```

Comment:
I would split this scenario into two:

1. to list all the content items
2. to click through an organisation

I will allow it to grow a bit easier in time.